### PR TITLE
FOCUS #75: Currency Code attribute and related content

### PR DESCRIPTION
**Pull request by udam-f2**
Currency Code attribute related changes.

Co-authored-by: Sanjna Srivatsa <ssrivatsa@vmware.com>
Co-authored-by: Eleni Rundle <elenirundle@google.com>

### CODE REVIEW

**Review by mikepolson-vmware — COMMENTED** [review]
Should we mention that the three-digit alpha code values must be from ISO-4217?  https://en.wikipedia.org/wiki/ISO_4217

File: specification/attributes/currency_code_format.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Currency Code Format
+
+Columns that contain currency information in cost data following a consistent format reduces friction for FinOps practitioners that consume the data for analysis, reporting, and other use cases.
+
+All currency related columns defined in the FOCUS specification MUST follow the formatting requirements listed below. Provider generated currency-related columns SHOULD adopt the same format requirements over time.
```
Comment (flanakin) [inline specification/attributes/currency_code_format.md:5]: nit: "FOCUS specification" is redundant. Should we use "schema" or just leave it out?
```suggestion
All currency related columns defined in the FOCUS schema MUST follow the formatting requirements listed below. Provider generated currency-related columns SHOULD adopt the same format requirements over time.
```

File: supporting_content/attributes/currency_code_format.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+| GCP      | Big Query Billing Export | Currency       |
+| Azure    | Cost details             | BilledCurrency |
+
+## Example usage scenarios
+
+Current values observed in billing data for various scenarios:
+
+| Provider | Data set                 | Example Value |
+|:---------|:-------------------------|:--------------|
+| AWS      | CUR                      | USD           |
```
Comment (flanakin) [inline supporting_content/attributes/currency_code_format.md:19]: uber-nit: I would suggest using a non-USD currency as the example.

**Review by flanakin — APPROVED** [review]

**Comment by udam-f2** [comment]
> Should we mention that the three-digit alpha code values must be from ISO-4217? https://en.wikipedia.org/wiki/ISO_4217

Hey @mikepolson-vmware, the requirements section says: "* Currency related columns MUST be represented as a three-letter alphabetic code as dictated in the governing document [ISO 4217:2015](https://www.iso.org/standard/64758.html)." Does that seem sufficient? That happens to be the only requirement right now for currency code but in the future we may have more so I want to as a pattern just refer to the requirements rather than naming individual ones. Let me know.

**Review by mike-finopsorg — APPROVED** [review]

**Pull request merged**